### PR TITLE
Add information of completion command in `wasp help` (#256)

### LINKED ISSUE #256: Make bash completion easier to discover and install
We have bash completion for Wasp thanks to @thvu11 !
In order to install it, user needs to run `wasp completion` to read the instructions and then run a couple of commands, which includes making decisions (decisions are hard!) like where to put the generated completion script.

There are two points here:
1. How will they know that bash completion exists and that they need to run `wasp completion`?
2. Can we make installation of bash completion easier, so they don't have to be making any decisions and it is as little steps as possible?

For discoverability, the easiest thing would be to add something to documentation (https://github.com/wasp-lang/wasp/tree/master/web/docs) -> we should certainly do that. We might also want to add info about `completion` command to the help that is printed when you type just `wasp` in terminal.

Next, we might want to inform users about this during the installation, in install script (https://github.com/wasp-lang/get-wasp/blob/master/installer.sh) -> something like "We have support for bash completion, run `wasp completion` to learn more!". Or, we could even add step to install script that asks: do you want us to install bash completion for you? Y/N? and if Y we do it automatically.

Making installation simpler -> I think one big step would be telling users where to put the completion script. For me personally it was not easy to figure out where to put it. We could at least recommend them some paths. Next step would be for us to make that decision -> we could maybe put the script at the same place where we install wasp stuff (binary, data)?
Also, we could even add that `source ...` line to the bash_profile/rc! I know some tools like `nvm` and `ruby` add stuff like that on their own.

We have seen **netlify** doing some nice things with their CLI, so it would probably be smart to again check how they do these things and maybe do it same as them, if we like what they do.

### PR DESCRIPTION
**Pull request by h4r1337**
# Description

This commit adds an addition information about the existence of the `wasp completion` command in the output of `wasp help` 

Issue #256 

## Type of change

Please select the option(s) that is more relevant.

- [ ] Code cleanup
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: waspc/cli/exe/Main.hs [hunk 1]
```diff
@@ -92,6 +92,9 @@ printUsage =
         cmd "    new <project-name>    Creates new Wasp project.",
         cmd "    version               Prints current version of CLI.",
         cmd "    waspls                Run Wasp Language Server. Add --help to get more info.",
+        title "  COMPLETION",
```
Comment (Martinsos) [inline waspc/cli/exe/Main.hs:95]: Maybe let's just move it under GENERAL, I don't think we need special title for it.
Also, I wonder if we should skip `completion:generate`? I believe that one is elaborated upon in `completion` command. Just to make it a bit less crowded here, I don't think we want completion taking too much focus.

**Review by Martinsos — COMMENTED** [review]
Left a tiny comment, let's take care of that and merge!

Oh btw you added statement "fix <issue number>", but that issue is much wider (although it includes this fix), so let's remove that statement or Github will close this issue which we don't want.

**Comment by h4r1337** [comment]
Hey @Martinsos fixed that. Is everything good now?

**Comment by Martinsos** [comment]
Great, merged! Thanks @h4r1337 !!!

**Pull request merged**